.NET/Microsoft.Recognizers.Definitions.Common/English/TimeZoneDefinitions.cs

```diff
@@ -35,6 +35,7 @@ public static class TimeZoneDefinitions
             @"ART",

```

Comment:
Can you sort this list alphabetically? It will greatly help spotting missing/incorrect items.